### PR TITLE
Add question issue template

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

**What this PR does / why we need it**:

Now users always mark any question as bug, it would mislead anyone who want to fix bugs.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+```
+</details>
+
+<details><summary>Kernel </summary>
+```console
+$ uname -a
+# paste output here
+```
+</details>
+
```
Comment (GsssC) [inline .github/ISSUE_TEMPLATE/question.md:101]: repetition
```suggestion

```

File: .github/ISSUE_TEMPLATE/question.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
```
Comment (GsssC) [inline .github/ISSUE_TEMPLATE/question.md:62]: repetition
```suggestion
```

**Comment by GsssC** [comment]
look good to me, I also think we need an kind of issue to submit question!
👍

File: .github/ISSUE_TEMPLATE/question.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Anything else we need to know?**:
+
+**Environment**:
+<details><summary>Kubernets Version</summary>
+```console
```
Comment (fisherxu) [inline .github/ISSUE_TEMPLATE/question.md:19]: Looks like the ` ```console` doesn't work within the `<details>`? 
image

Comment (daixiang0) [reply .github/ISSUE_TEMPLATE/question.md:20]: Fix it, in fact it needs extra empty lines.

**Comment by fisherxu** [comment]
Looks fine :)  But the template file seems a bit long for users to write.. And when we want to see the version of components, we need to click them one by one?

**Comment by daixiang0** [comment]
How about only saying os arch and kernel version for each component, and put the version of components outside, then we can easily to get version info.

**Comment by kevin-wangzefeng** [comment]
Do we really want to receive questions on github?
Or probably just for a period of time until we find a better place.
I'm just afraid of messing up the issues list.

**Comment by kevin-wangzefeng** [comment]
And we do need to add some guide to avoid issues created with "kind/bug" but are actually questions.

**Comment by daixiang0** [comment]
In k8s, the question template is just a entry to tip user that they should go to other places to ask questions.

For now, we still need to split bug and question, the first step is to add a question template.

Then, we can learn k8s, only keep question template as a entry.

**Pull request merged**